plugins/auth/index.js

```diff
@@ -112,6 +116,19 @@ exports.register = (server, options, next) => {
             }
         });
 
+        server.auth.strategy('auth_token', 'bearer-access-token', {
+            accessTokenName: 'access_token',
+            allowCookieToken: false,
+            allowQueryToken: true,
+            validateFunc: (token, cb) => {
+                if (token !== pluginOptions.temporaryAccessToken) {
+                    return cb(null, false, {});
+                }
+
+                return cb(null, true, { username: '???????', scope: ['user'] });
```

Comment:
* Would it be appropriate to use our team user?
* What would be an appropriate scope for them?